README.md

```diff
@@ -14,7 +14,22 @@ imports immediately. But this becomes tedious, especially if you use explicit im
 
 _Importify_ can remove unused imports automatically.
 
-In the future, we plan for _Importify_ to be able to add used symbols to imports automatically, so that you won't have to manage imports manually at all. To speed up _Importify_ runs, we will implement a cache server that caches each _Hackage_ package. You will be able to upload caches for your projects there as well. Also on the roadmap is functionality to convert imports between _implicit_ and _explicit_, and between _qualified_ and _unqualified_ forms. Finally, we will be adding functionality to resolve merge conflicts in import section automatically.
+In the future, we plan for _Importify_ to be able to:
+
+ + Add missing imports automatically, so you won't have to manage
+   imports manually at all.
+ + Implement a cache server with the following features:
+   + Download caches for Hackage packages to speed up _Importify_ runs.
+   + Upload your caches for yet-to-be-published FOSS projects to
+     make it easier to collaborate.
+   + Query mappings from any module of every package to symbols
+     exported by it to write your refactoring tools.
+ + Convert imports between _implicit_ and _explicit_, and between
+   _qualified_ and _unqualified_ forms.
+ + Add symbols that are not present in the import section of a module.
+ + Resolve merge conflicts in import section automatically. See an example below.
+
+![Merge conflict in import section](http://i.imgur.com/97YVCFk.png)
```

Comment:
Inserted image is very big. It will take major part of user attention despite the fact that it's not so important. I suggest to resize this image and decrease its size. I never did this before by myself but you can try next options:

1. Specify size directly in markdown: https://gist.github.com/uupaa/f77d2bcf4dc7a294d109
2. Use `HTML` only for image insted of markdown: https://stackoverflow.com/questions/24383700/resize-image-in-the-wiki-of-github-using-markdown
3. Resize image manually using some editor or online web service.

If non of the options work I will try to do it by myself. I have any questions don't shy to ask me. And don't worry about spoiling commit history with multiple non-working solutions. I can prettify commit history later if you don't feel confident with such actions.